### ISSUE TITLE
Ao entrar no INTEGRASUS, a tela inicial (verde) está cortada à direita

### ISSUE THREAD

**Issue by drmarcos** [EscolaDeSaudePublica/isus-app#254]
1, Ao entrar no INTEGRASUS, a tela inicial (verde) está cortada à direita, aparecendo apenas “INTEGRAS”, assim como a segunda caixa laranja. Estou no IOS em um iPhone 7. 

2. Aqui, o botão ENVIAR, está escondido sob o teclado, ele recolhe ao tentar descer mais a tela. Precisei tocar fora desse texto para liberar a tela inteira. Não me pareceu intuitivo.

https://felicilab.freshdesk.com/a/tickets/42

**Comment by matheuscpl** [comment]
Lembrando que o IntegraSUS não é mantido pela ESP mas pela SESA, esse erro ocorre por uma falha na responsividade do serviço deles. O erro ocorre não apenas no iSUS mas em qualquer acesso a partir de dispositivos móveis.
Acredito que a tarefa aqui é entrar em contato com o time de inovação de lá para ver o problema...

**Comment by drmarcos** [comment]
É um iFrame, é isso?

**Comment by matheuscpl** [comment]
@drmarcos Sim, como algo do tipo. No React Native é chamado de **webview** mas o comportamento é bem semelhante ao iframe.

**Comment by laurapmelo** [comment]
>>>>> 1, Ao entrar no INTEGRASUS, a tela inicial (verde) está cortada à direita, aparecendo apenas “INTEGRAS”, assim como a segunda caixa laranja. Estou no IOS em um iPhone 7.

Vou considerar esta como resolvida, já que está funcionando corretamente agora, certo?
Vou adaptar o card para a segunda demanda.

**Comment by ItaloRod** [comment]
tarefa pausada durante o processo de estruturação das branches.

**Comment by kurjes** [comment]
@nicoledbianchin falou que o scroll de página no android não funciona, então a sugestão de como solucionar esse comportamento não cabe. Conversando com @rodrigoplimafreire ele informou que o comportamento de teclado cobrir o botão enquanto o usuário está digitando é normal, então fica em aberto se esse card é um problema real.

**Comment by kurjes** [comment]
Durante o desenvolvimento tivemos problemas com o campo de autocomplete e atualmente temos em produção um campo que não corresponde a experiência de usuário pensada. É necessária uma priorização do card e depois uma ideação de UX para que o usuário possa ter uma melhor experiência. (@rodrigoplimafreire @uira) 

pareando com @daianefreira

**Comment by rodrigoplimafreire** [comment]
Fiz três sugestões, mas este ponto precisamos discutir.
![image](https://user-images.githubusercontent.com/46686642/96281741-25563700-0fb0-11eb-9e83-033c832fc82c.png)

**Comment by rodrigoplimafreire** [comment]
Vejamos o comportamento do _Facebook_. Na mesma seção de **Fale Conosco**, ou qualquer que seja o formulário, o comportamento previsto pelo _Facebook_ é de colocar os botões de **Enviar** sempre acima, na _Barra do App (Surface)_, facilitando a experiência do usuário quando ele clicar no campo que irá preencher. Se formos adotar este comportamento, precisaremos alterar todas as nossas telas que trabalham com preenchimento de formulário e precisam de um botão de **Enviar ou Salvar**, por exemplo. É uma solução "drástica", mas sensata. 
![facebook](https://user-images.githubusercontent.com/46686642/96281900-59c9f300-0fb0-11eb-8657-e5bcf3b9087e.jpg)

**Comment by daianefreira** [comment]
Necessário conversa para decidir por que caminho seguimos.

**Comment by daianefreira** [comment]
@wforte88 Favor verificar com qual experiência seguimos...

**Comment by wforte88** [comment]
Tarefa aguarda definição de novos padrões (facelift) para uniformizar a experiência.

**Comment by MayaraMonte** [comment]
Protótipo atualizado: https://www.figma.com/file/FLKurtBDSBYAiYcS5xAPYC/?node-id=5610%3A8

**Comment by ruigmoreno** [comment]
## Análise de aplicativos mobile do mercado

### Aplicativo Bradesco

https://user-images.githubusercontent.com/25174912/107673179-82b77780-6c74-11eb-8bc5-a67d011a8116.mp4

### Aplicativo Google

https://user-images.githubusercontent.com/25174912/107672959-408e3600-6c74-11eb-8124-21d57c41e059.mp4

### Aplicativo Nubank

https://user-images.githubusercontent.com/25174912/107673075-64517c00-6c74-11eb-8f7c-05abcf7748be.mp4

### Conclusão

De três aplicações bancárias analisadas, apenas uma trata essa questão do botão ficar coberto. Outra aplicação que trata essa questão é a de criação de uma conta google. Na aplicação google, analisa-se um comportamento parecido com o padrão adotado pelo iSUS, porém não se polui muito a tela de inputs (entradas) de dados.  Deve-se analisar o trade-off entre a experiência de usuário e o valor de negócio que essa atividade irá retornar para os clientes, tendo em vista o tempo e o esforço para dedicar à analisar o tratamento desta tarefa.
